server/graphql/v2/interface/Account.js

```diff
@@ -562,6 +571,28 @@ export const AccountFields = {
       return get(collective.data, 'categories', []);
     },
   },
+  updates: {
+    type: UpdateCollection,
+    args: {
+      limit: { type: GraphQLInt },
+      offset: { type: GraphQLInt },
```

Comment:
Can be replaced by:

```suggestion
      ...CollectionArgs,
```

Benefits:
- Add default value on `limit` => we prefer that people don't fetch all expenses at once
- Add `description`s (better docs)

Reply:
1. Do you have an example of this in use?
2. Why do I keep seeing the word 'expenses' used with Updates? How are they related? I don't see the connection.

Reply:
https://github.com/opencollective/opencollective-api/blob/5cd771d0436ab06fe5ef34eaca7bd7ef2ab517ec/server/graphql/v2/query/TransactionsQuery.ts#L14